### PR TITLE
Disclaimer

### PR DESCRIPTION
**Pull request by pataruco**
### Name of the story

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/recBaAdtosmE3lNNW?blocks=hide)

### Acceptance Criteria

- [ ] Show once per visit underneath the search result page copy on the search results page
- [ ] Include the checkbox 
- [ ]  Hide search results until checkbox has been checked

### Testing information

- Open the site in the browser of your preference, do a search and you should see a disclaimer box before it shows search results
- Check on the checkbox and the disclaimer should disappear and show the search results
- I

### Pre-flight Checklist

- ~Tests~
- [x] DUXD review approval
- [x] Accessibility Reviewed
- [x] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: medicines/web/hooks/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+export const useSessionStorage = (key: string, initialValue: any): any => {
+  const [storedValue, setStoredValue] = useState(() => {
+    try {
+      const item = window.sessionStorage.getItem(key);
+      return item ? JSON.parse(item) : initialValue;
+    } catch (error) {
+      // tslint:disable-next-line: no-console
+      console.error(error);
```
Comment (m-doughty) [inline medicines/web/hooks/index.tsx]: Is this intentional?

File: medicines/web/hooks/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  const setValue = (value: any) => {
+    try {
+      const valueToStore =
+        value instanceof Function ? value(storedValue) : value;
+      setStoredValue(valueToStore);
+      window.sessionStorage.setItem(key, JSON.stringify(valueToStore));
+    } catch (error) {
+      // tslint:disable-next-line: no-console
+      console.error(error);
```
Comment (m-doughty) [inline medicines/web/hooks/index.tsx]: Is this intentional?

File: medicines/web/hooks/index.tsx (hunk 1 above)
Comment (pataruco) [reply medicines/web/hooks/index.tsx]: yes

File: medicines/web/hooks/index.tsx (hunk 2 above)
Comment (pataruco) [reply medicines/web/hooks/index.tsx]: yes

**Pull request merged**